src/Card/Card.js

```diff
@@ -1,12 +1,26 @@
+import cl from 'classnames';
 import './Card.scss';
 
-function Card({ className, caption, content }) {
+function Card({ className, caption, content,
+                checkbox = false, selected = false,
```

Comment:
I think "checkbox" property is needless.

Reply:
How could we choose between selectable and not selectable cards then?

Reply:
Can you please provide me text which describes that card may not be selectable?